src/modules/premium-schedules/dto/create-premium-schedule.dto.ts

```diff
@@ -0,0 +1,72 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsDateString, IsInt, IsNotEmpty, IsNumber, IsOptional, IsString, Length, MaxLength } from 'class-validator';
+
+export class CreatePremiumScheduleDto {
+  @IsInt()
+  @IsNotEmpty()
+  @ApiProperty({ example: 10588388, description: 'Can be BS or EW' })
+  readonly facilityURN: number;
+
+  @IsString()
+  @IsNotEmpty()
+  @Length(2, 2)
+  @ApiProperty({ example: 'BS', description: 'Can be BS or EW' })
+  readonly productGroup: string;
+
+  @IsNumber()
+  @IsNotEmpty()
+  @ApiProperty({
+    example: 1,
+    description: 'Premium type concerns how we are being paid... In Arrears, In advance or one lump sum. See MASTER_DATA.dbo.WF_PREMIUM_TYPE',
```

Comment:
@IgnatG mentioned that we might want to allow teams across UKEF to use this documentation - will `MASTER_DATA.dbo.WF_PREMIUM_TYPE` mean much to them?

Reply:
I checked table and very few premium types are available. Added possible value list to the description.